src/engraving/dom/beambase.h

```diff
@@ -46,7 +46,7 @@ struct BeamFragment {
     double py2[2];
 };
 
-enum class ChordBeamAnchorType {
+enum class ChordBeamAnchorType : char {
```

Comment:
It's better to explicitly specify `signed` or `unsigned` `char`, it's just that `char` has a different representation on different platforms.

Reply:
Done!